kuzzleio/kuzzle-admin-console
# Kuzzle multiversion proof-of-concept

## What does this PR do ?

Implements the proof-of-concept showing that the same instance of the Admin Console can connect to a Kuzzle v1.x and a Kuzzle v2.x (issue #682)

### How should this be manually tested?
Be sure to have 2 instances of Kuzzle, one v1 and one v2. Create two different connections, one for each backend and specify the correct version for each backend. Verify you can connect, login and see the list of the indexes

### Important note
This is still in the POC phase, this the implementation is lacking many features. Don't expect the Admin Console to work beyond the scope of the POC, i.e. connecting, logging-in and listing the indexes.
This (very succint) work is meant to help us decide whether we take this path or not.

### TODO
- [x] Tests v2
- ~Tests v1~
- [x] Show consistent feedback when connecting to a Kuzzle of the wrong version
- ~CI with stages for v1 and v2~

### Boyscout
- Got rid of `vuex/common/crudlDocument`
